docs: template_updates

Jen-Uno opened the pull request:
GitHub Issue (If applicable): #

<!-- Link to the relevant GitHub issue if applicable. All PRs should be associated with an issue (GitHub or internal). -->

## PR Type

What kind of change does this PR introduce?  
<!-- Please uncomment one or more that apply to this PR: -->

- [ ] Bugfix  
- [ ] Feature  
- [ ] Code style update (formatting)  
- [ ] Refactoring (no functional changes, no API changes)  
- [ ] Build or CI related changes  
- [ ] Documentation content changes  
- [ ] Project automation  
- [ ] Other (please describe):  

## What is the current behavior?

<!-- Describe the current behavior that is being modified, or link to a relevant issue. -->

## What is the new behavior?

<!-- Describe the new behavior after your modifications. -->

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Documentation has been added/updated  
- [ ] Associated with an issue (GitHub or internal)

## Additional Information

<!-- Provide any additional information or context, if necessary. -->

Jen-Uno committed:
docs: template_updates

vatsashah45 commented on .github/ISSUE_TEMPLATE/enhancement.yml:
```diff
@@ -18,10 +18,12 @@ body:
     attributes:
       label: Which tool(s) are affected?
-      description: ([Hot Design®](https://aka.platform.uno/hot-design), [Hot Reload](https://aka.platform.uno/hot-reload), or [Design-to-Code](https://aka.platform.uno/Design-to-Code))
+       description: ([Hot Design®](https://aka.platform.uno/hot-design), [Hot Design® Agent](https://aka.platform.uno/hot-design-agent), [Uno MCPs](https://aka.platform.uno/using-uno-mcps), [Hot Reload](https://aka.platform.uno/hot-reload), or [Design-to-Code](https://aka.platform.uno/Design-to-Code))
```
Remove whitespace at the beginning of the sentence

vatsashah45 commented on .github/ISSUE_TEMPLATE/documentation-request.yml:
```diff
@@ -12,10 +12,12 @@ body:
     attributes:
       label: Which tool(s) are affected?
-      description: ([Hot Design®](https://aka.platform.uno/hot-design), [Hot Reload](https://aka.platform.uno/hot-reload), or [Design-to-Code](https://aka.platform.uno/Design-to-Code))
+       description: ([Hot Design®](https://aka.platform.uno/hot-design), [Hot Design® Agent](https://aka.platform.uno/hot-design-agent), [Uno MCPs](https://aka.platform.uno/using-uno-mcps), [Hot Reload](https://aka.platform.uno/hot-reload), or [Design-to-Code](https://aka.platform.uno/Design-to-Code))
```
Remove whitespace at the beginning of the sentence

vatsashah45 commented on .github/ISSUE_TEMPLATE/feedback.md:
```diff
@@ -10,7 +10,7 @@ Here are some ideas to guide your feedback:
 ### General Feedback
 - Which version of Uno Platform (Uno.Sdk) are you using, on which platforms, and with which IDE?
-- What do you love about Uno Platform Studio and its related tools (Hot Design™, Hot Reload, Design-to-Code)?
+- What do you love about Uno Platform Studio and its related tools (Hot Design™, Hot Design™ Agent, Uno MCPS, Hot Reload, Design-to-Code)?
```
```suggestion
- What do you love about Uno Platform Studio and its related tools (Hot Design™, Hot Design™ Agent, Uno MCPs, Hot Reload, Design-to-Code)?
```

Jen-Uno committed:
chore: Fix formatting of description in documentation request template

Jen-Uno committed:
chore: Update .github/ISSUE_TEMPLATE/feedback.md

Co-authored-by: Vatsa Shah <vatsashah45@gmail.com>

Jen-Uno committed:
chore: Modify tools dropdown description in enhancement template

Updated the description for the tools dropdown in the enhancement issue template.
